R_gnss: [0.09843, 0, 0, 0.0984]
R_imu: 1.894e-06
R_tach: 5.538e-06 #hasil itung
initial_estimate_covariance_P: [10, 0, 0, 0, 0, 0, 10, 0, 0, 0, 0, 0, 10, 0,
  0, 0, 0, 0, 10, 0, 0, 0, 0, 0, 1] #pernah tuning yg 4 awal tapi kayanya masi ngaco
kappa: -2 #paper
process_noise_covariance_Q: [10, 0, 0, 0, 0, 0, 10, 0, 0, 0, 0, 0, 10, 0, 0, 0, 0, 0,
  10, 0, 0, 0, 0, 0, 1]
